notebook/autogen_agent_web_info.ipynb

```diff
@@ -48,7 +48,7 @@
    },
    "outputs": [],
    "source": [
-    "# %pip install flaml[autogen]==2.0.0rc3"
+    "# %pip install flaml[autogen]~=2.0.0rc4"
```

Comment:
The initiation of chat in the example in this notebook should be updated to `user.initiate_chat(...)`?